src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -9,19 +9,20 @@
     <PackageTargetRuntime>$(MinOSForArch)-$(PackagePlatform)</PackageTargetRuntime>
   </PropertyGroup>
   <ItemGroup>
-    <ArchitectureSpecificNativeFile Include="$(BinDir)clretwrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)coreclr.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)dbgshim.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordaccore.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)clretwrc.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)coreclr.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)dbgshim.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscordaccore.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscorrc.debug.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscorrc.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)sos.dll" />
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll" />
     <ArchitectureSpecificNativeFile Include="$(BinDir)System.Private.CoreLib.ni.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorrc.debug.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)sos.dll" />
     <ArchitectureSpecificLibFile Include="$(BinDir)System.Private.CoreLib.dll" />
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll" />
```

Comment:
I see! I expected them to be in the same relative directory in the symbol package as the matching dll/exe in the library package, but it doesn't matter to the service where they end up so that's fine. (I see it simplifies the code further down to keep them all in the same place.)